code/ATMOSPHERICS/components/binary_devices/circulator.dm

```diff
@@ -75,18 +75,18 @@
 
 /obj/machinery/atmospherics/binary/circulator/update_icon()
 	icon_state = anchored ? "circ-assembled" : "circ-unassembled"
-	cut_overlays()
+	overlays.Cut()
```

Comment:
Same as before with these kinda things sadly

Reply:
yeah, but in the omni filters they need to be the way they are or they do not work at all
but yes, on the circulator changing still works